.github/workflows/master-push.yml

```diff
@@ -70,24 +68,22 @@ jobs:
       name: Package examples
       needs: prepare
       steps:
-          - name: Checkout
-            uses: actions/checkout@v4
-          - name: Prepare examples for packaging
-            run: sh -x build.sh release-package-examples
-          - name: Upload examples to artifacts library
-            uses: actions/upload-artifact@v4
-            with:
-              path: realm-examples.zip
-              name: realm-examples
+        - uses: actions/checkout@v4
+        - name: Prepare examples for packaging
+          run: sh -x build.sh release-package-examples
+        - name: Upload examples to artifacts library
+          uses: actions/upload-artifact@v4
+          with:
+            path: realm-examples.zip
```

Comment:
My mistake as well, but to maintain same order it should be name and path instead of path and name